style.css

```diff
@@ -0,0 +1,138 @@
+.container {
+    display: flex;
+    margin-left: auto;
+    margin-right: auto;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+
+
+
+}
+.header {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: yellow underline;
+    font-size: 50px;
+    height: 300px;
+    width: 450px;
+    padding-bottom: 200px;
+    color: black;
+   
+}
+
+.pic-trio {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-content: space-between;
+    height: 500px;
+    width: 1250px;
+    padding: 200px;
+  
+}  
+
+
+.container2 {
+    display: flex;
+    flex-direction: row;
+    align-content: center;
+    padding: 50px;
+}
+.quote-box {
+    display: flex;
+    flex-direction: column;
+    width: 1250px;
+    height: 100px;
+  
+}
+
+.container3 {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    padding: 200px;
+}
+.content-1{
+    display: flex;
+    flex-direction: column;
+    text-align: left;
+    flex-flow: wrap;
+    text-justify: auto;
+    height: 1250px;
+    width: 750px;
+    padding-right: 100px;
+   
+}
+.content-2{
+    display: flex;
+    flex-direction: column;
+    text-align: left;
+    flex-flow: wrap;
+    text-justify: auto;
+    height: 1250px;
+    width: 750px;
+    padding-left: 100px;
+    
+  
+}
+
+.sidebar {
+    display: flex;
+    flex-direction: column;
+    text-align: left;
+    flex-flow: wrap;
+    text-justify: auto;
+   width: 250px;
+   height: 1500px;
+   padding-left: 150px;
+   font-size: 15px !important;
+   
+}
+.container5 {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    padding: 200px;
+}  
+
+.footer {
+    display: flex;
+    width: 1000px;
+    height: 200px;
```

Comment:
For sections like this, it might be more practical to use % or vw/vh for your sizes. Looking at this on a 27in monitor or a 13in monitor is going to give you drastically different results if you're using a defined size such as pixels

This comment is also relevant for most other sections in your code that you define a size